CommentingPractice/JavaScript Commit.js

```diff
@@ -0,0 +1,18 @@
+/**This is my first comment
+ * Represents a car
+ * @constructor
+ * @param {int} num_doors
+ * @param {string} brand
+ * @param {string} color
+ */
+
+ function Car (num_doors,brand,color)
+ {
+ 	return "The car has " + string(num_doors) + " and is a " + color + brand + ".";
+ }
+ for (var x = 0; x<10;x++)
+ {
+  console.log(x);
+ }
+ console.log("\n");
+ console.log(Car(4,"Toyota","red"));
```

Comment:
Take a look at this resource: https://www.w3schools.com/js/js_classes.asp

Try to work what you have into something like what you see on that site.